Task_12/api/views/client_api.py

```diff
@@ -0,0 +1,70 @@
+import json
+from flask import request, jsonify
+
+from truck_delivery_rayus007.client import Client
+from truck_delivery_rayus007.client_manager import ClientManager
+
+
+def list_clients():
+    """
+    list_clients --> lists all the clients
+    :return: clients(dict): Dictionary of Clients
+    """
+    connector = ClientManager()
+    result = connector.get_all()
+    if result:
+        return jsonify(result)
+    else:
+        return jsonify({"message": "Clients Not found"}), 500
```

Comment:
If no clients are found, I don't think you should be returning a 500 code, maybe it would be better to return just an empty list

Reply:
Updated to return empty list.